about/press.html

```diff
@@ -6,79 +6,66 @@
-              </object>
-            </div>
-            <ul>
-                <li><a href="http://railsgirlssummerofcode.org/press/release17june/">Press Release 17 June, 2013: International Rails Girls Summer of Code Gets Women into Open Source</a></li>
-                <li><a href="http://blog.railsgirls.com/post/51878746081/what-exactly-is-open-source-and-why-does-it-need">What exactly is “Open Source” (and why does it need more women)?</a></li>
-                <li><a href="http://railsgirls.com/press">Rails Girls Press</a></li>
-            </ul>
-        <h3>Photos</h3>
-        <p>(click on a photo for a high resolution version)</p>
+  <div class="wrapper clearfix">
+    <article class="content">
+      <h1>{{page.title}}</h1>
+      <h3>About Rails Girls Summer of Code</h3>
+      <p>
+      Rails Girls Summer of Code is a scholarship that funds women around the world to work full-time over the summer on open-source software projects. Modelled after Google Summer of Code, but with a focus on beginners and women wanting to go into programming full-time, Rails Girls Summer of Code aims to get women involved in open-source and is the first ever event of its kind in the world.
+      </p>
 
-        <a target="_blank" href="https://f.cloud.github.com/assets/1711357/653139/5e1f3188-d4ce-11e2-9f96-9518ebb7f721.jpg"><img src="https://f.cloud.github.com/assets/1711357/653139/5e1f3188-d4ce-11e2-9f96-9518ebb7f721.jpg" title="Coding at Rails Girls Kyoto" height="150px"></a>
+      <p>
+      Since 2013, over 580 women all over the world have applied and up to this date, RGSoC has supported 106 students — with a total of 35 sponsored teams and 20 volunteering teams. Recently, a survey conducted by RGSoC revealed that 90% of the students are still working in tech and 8% have founded their startups. 55% of those students still regularly contribute to Open Source projects.
+      </p>
 
-        <a target="_blank" href="https://f.cloud.github.com/assets/1711357/653102/70f37298-d4cd-11e2-9a44-ab9683bf9521.jpg"><img src="https://f.cloud.github.com/assets/1711357/653102/70f37298-d4cd-11e2-9a44-ab9683bf9521.jpg" title="Linda Liukas, Co-Founder of Rails Girls, at Rails Girls Helsinki" height="150px"></a>
+      <p>
+      This year, over 90 teams applied for the program, almost doubling RGSoC 2015’s record! So far, 44 Open Source projects were accepted for this edition and 12 teams will be funded. All selected teams, including volunteer teams, will be announced at the beginning of June. Meanwhile, RGSoC has launched the <a href="http://railsgirlssummerofcode.org/blog/2016-04-22-thank-you-lets-diversify-tech">#diversifytech</a> campaign to thank all of the people who have contributed to the program.
```

Comment:
sound great @carpodaster! Good point. :) I'll do this right now if you're cool with that